### PR TITLE
Update travel.md (add private car reimbursement)

### PR DESCRIPTION
**Pull request by fdelin**
Added section on using your own car

### CODE REVIEW

File: guides/compensation/expenses/travel.md [hunk 1]
```diff
@@ -2,15 +2,29 @@
...
 - Taxi home after working late
 - Travelling to work at a client site outside of the region of your Made Tech office
 
 
+## Using your own car
+You can use your private car where you cannot or it's not practical to use public transportation to travel for business. The current reimbursement rate as prescribed by HMRC is 45p per mile (up to 10k miles).
+
+When claiming your mileage, please use a journey planner such as Google Maps to get the round-trip distance and include a screenshot when expensing. 
+
+To allow MadeTech to claim the VAT on the journey’s fuel element, please can you provide the following:
```
Comment (yusufsheiqh) [inline guides/compensation/expenses/travel.md:17]:
```suggestion
To allow Made Tech to claim the VAT on the journey’s fuel element, please provide the following:
```

File: guides/compensation/expenses/travel.md [hunk 2]
```diff
@@ -2,15 +2,29 @@
...
+- A petrol receipt (showing VAT) dated max 2 months before the journey.
+- Add to the expense description the engine size of the car and if it's petrol or diesel.
+
+This part is optional and the expense total should always be miles*applicable reimbursement rate.
+
+If using your own car, you must ensure that adequate insurance and breakdown cover is in place. MadeTech will not assume any responsibility for breakdowns, fines, accidents or any damage. If unsure of any of the above, please contact Finance for additional guidance.
```
Comment (yusufsheiqh) [inline guides/compensation/expenses/travel.md:23]:
```suggestion
If using your own car, you must ensure that adequate insurance and breakdown cover is in place. Made Tech will not assume any responsibility for breakdowns, fines, accidents or any damage. If unsure of any of the above, please contact the Finance team for additional guidance.
```

File: guides/compensation/expenses/travel.md [hunk 3]
```diff
@@ -2,15 +2,29 @@
...
+
+
 ## Travel that is not covered
 
-Where you're employed as a member of staff in a Made Tech office, your travel to that office or to a client site within that metropolitan area is not be covered.
+Where you're employed as a staff member in a Made Tech office, your regular travel to that office or to a client site within that metropolitan area is not be covered.
```
Comment (yusufsheiqh) [inline guides/compensation/expenses/travel.md:28]:
```suggestion
Where you're employed as a staff member in a Made Tech office, your regular travel to that office, or to a client site, within that metropolitan area is not covered.
```

**Review by duncanmadetech — APPROVED** [review]
LGTM

File: guides/compensation/expenses/travel.md [hunk 4]
```diff
@@ -2,15 +2,29 @@
...
-Some examples of expensable work travel:
+Some examples of work travel you can expense:
 
 - Public transport / taxi to a client site for a meeting
 - Taxi home after working late
 - Travelling to work at a client site outside of the region of your Made Tech office
 
 
+## Using your own car
+You can use your private car where you cannot or it's not practical to use public transportation to travel for business. The current reimbursement rate as prescribed by HMRC is 45p per mile (up to 10k miles).
```
Comment (yusufsheiqh) [inline guides/compensation/expenses/travel.md:13]: I think a link to the HMRC web page would be good here. I assume it's the following link - https://www.gov.uk/expenses-and-benefits-business-travel-mileage/rules-for-tax

If we do add the link, maybe the heading for this section can be updated to say "Using your own vehicle" instead(?).

Comment (fdelin) [reply guides/compensation/expenses/travel.md]: Great idea. Updated.

**Pull request merged**